tests/benchmarks/micro_benchmarks/test_ort_inference_performance.py

```diff
@@ -0,0 +1,63 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+"""Tests for ort-inference benchmark."""
+
+import shutil
+from pathlib import Path
+from unittest import mock
+
+import torch
+import torchvision.models
+
+from tests.helper import decorator
+from superbench.benchmarks import BenchmarkRegistry, Platform, Precision, BenchmarkType, ReturnCode
+from superbench.benchmarks.micro_benchmarks.ort_inference_performance import ORTInferenceBenchmark
+
+
+@decorator.cuda_test
+@decorator.pytorch_test
+@mock.patch('torch.hub.get_dir')
+@mock.patch('onnxruntime.InferenceSession.run')
+def test_ort_inference_performance(mock_ort_session_run, mock_get_dir):
```

Comment:
is `mock_ort_session_run` actually used?

Reply:
Yes, it is used.